src/Home/index.tsx

```diff
@@ -82,198 +86,263 @@ const Home: FunctionComponent = () => {
-  const topIconAccessibilityLabel = appIsActive
-    ? t("home.status_icon_active_label")
-    : t("home.status_icon_inactive_label")
-  const headerText = appIsActive
+  const handleOnPressCallEmergencyServices = () => {
+    Linking.openURL(`tel:${emergencyPhoneNumber}`)
+  }
+
+  const statusBackgroundColor = exposureDetectionStatus
+    ? Colors.success10
+    : Colors.danger10
+  const statusBorderColor = exposureDetectionStatus
+    ? Colors.success100
+    : Colors.danger100
+  const statusIcon = exposureDetectionStatus
+    ? Icons.CheckInCircle
+    : Icons.XInCircle
+  const statusIconFill = exposureDetectionStatus
+    ? Colors.success100
+    : Colors.danger100
+  const statusText = exposureDetectionStatus
     ? t("home.bluetooth.tracing_on_header")
     : t("home.bluetooth.tracing_off_header")
-  const subheaderText = () => {
-    if (appIsActive) {
-      return t("home.bluetooth.all_services_on_subheader", {
-        applicationName,
-      })
-    } else {
-      return isLocationRequired
-        ? t("home.bluetooth.tracing_off_subheader_location")
-        : t("home.bluetooth.tracing_off_subheader")
-    }
-  }
+  const actionText = exposureDetectionStatus
+    ? t("exposure_scanning_status.learn_more")
+    : t("exposure_scanning_status.fix_this")
 
-  const SettingsButton = () => {
-    return (
-      <TouchableOpacity
-        style={style.settingsButtonContainer}
-        accessible
-        accessibilityLabel={t("home.open_settings")}
-        onPress={handleOnPressSettings}
-      >
-        <SvgXml
-          xml={Icons.Gear}
-          width={Iconography.small}
-          height={Iconography.small}
-          fill={Colors.white}
-        />
-      </TouchableOpacity>
-    )
+  const statusContainerStyle = {
+    ...style.statusContainer,
+    backgroundColor: statusBackgroundColor,
+    borderColor: statusBorderColor,
   }
 
   return (
     <>
-      <StatusBar backgroundColor={Colors.gradient100Light} />
+      <StatusBar backgroundColor={Colors.primaryLightBackground} />
       <ScrollView
         style={style.container}
         contentContainerStyle={style.contentContainer}
       >
-        <View style={style.topScrollViewBackground} />
-        <GradientBackground gradient={Colors.gradient100} angleCenterY={1}>
-          <View style={style.topContainer}>
-            <SettingsButton />
-            <View style={style.topIconContainer}>
+        <Text style={style.headerText}>{t("screen_titles.home")}</Text>
+        <TouchableOpacity
+          style={style.settingsButtonContainer}
+          accessible
+          accessibilityLabel={t("home.open_settings")}
+          onPress={handleOnPressSettings}
+        >
+          <SvgXml
+            xml={Icons.Gear}
+            width={Iconography.small}
+            height={Iconography.small}
+            fill={Colors.neutral100}
+          />
+        </TouchableOpacity>
+        <TouchableOpacity
```

Comment:
What do you think about `<ExposureStatusIndicator />` ?

Reply:
👍